packages/wrangler/src/d1/execute.tsx

```diff
@@ -228,16 +228,19 @@ async function executeLocally({
 	const localDB = getDatabaseInfoFromConfig(config, name);
 	if (!localDB) {
 		throw new Error(
-			`Can't find a DB with name/binding '${name}' in local config. Check info in wrangler.toml...`
+			`Couldn't find a DB with name/binding '${name}' in wrangler.toml`
 		);
 	}
 
-	const id = localDB.previewDatabaseUuid ?? localDB.uuid;
+	// const id = localDB.previewDatabaseUuid ?? localDB.uuid;
+	// TODO: use the `id` to move any existing SQL files/folders to `binding`
```

Comment:
@mrbbot is there an easy way to move any previously created local `miniflare-D1DatabaseObject` to the new folder here?

Reply:
Nevermind, I found `durableObjectNamespaceIdFromName`